src/olympia/amo/models.py

```diff
@@ -414,6 +391,11 @@ def update(self, **kw):
             models.signals.post_save.send(sender=cls, instance=self,
                                           created=False)
 
+    def save(self, **kwargs):
+        if hasattr(self._meta, 'translated_fields'):
+            save_translations(id(self))
```

Comment:
Actually I stole that from marketplace and it's not the case regarding `pre_save` - the problem is that `save` checks for any m2n relation to have the parent saved and only after that saves the actual model so we get an error.